chapters/adv_tfx/Custom_TFX_Components.ipynb

```diff
@@ -24,7 +25,7 @@
       "source": [
         "## Copy your notebook version\n",
         "\n",
-        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/googlecolab/colabtools/blob/master/notebooks/colab-github-demo.ipynb)"
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Building-ML-Pipelines/building-machine-learning-pipelines/blob/master/chapters/adv_tfx/Custom_TFX_Components.ipynb) "
```

Comment:
Looks good. But you always get redirected to the version on the master branch, e.g. in case of a PR it was a little bit confusing. You could change the link to: https://colab.research.google.com/github/Building-ML-Pipelines/building-machine-learning-pipelines
Then the user can select all notebooks and branch versions in the repo. Not perfect because you need an additional select, you can decide what you prefer.

Reply:
Thank you for your suggestion!